.github/workflows/automerge.yml

```
@@ -26,6 +26,31 @@ jobs:
             GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
             PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
 
+        - name: Fail workflow if PR is not mergeable
+          if: ${{ steps.isPullRequestMergeable.outputs.IS_MERGEABLE }}
```

Should this be ?

```suggestion
          if: ${{ steps.isPullRequestMergeable.outputs.IS_MERGEABLE == 'false' }}
```

Also do you know if this exit 1 will stop all the other jobs from running and so therefore we don't need the `if:` checks for `== 'true'`?

---

Fixed the conditional!

> Also do you know if this exit 1 will stop all the other jobs from running and so therefore we don't need the if: checks for == 'true'?

Yeah, I think this is true.